haiku-apps/beae/beae-1.2.recipe

```diff
@@ -4,9 +4,12 @@ in/out, analyzing, etc."
 HOMEPAGE="https://github.com/HaikuArchives/BeAE"
 COPYRIGHT="2003 Xentronix"
 LICENSE="BSD (3-clause)"
-REVISION="3"
-SOURCE_URI="$(HOMEPAGE)/archive/a90a9872420afe1a06537c798bfdfdc36d631323.tar.gz"
-SOURCE_DIR="beae-1.2"
+REVISION="4"
+SOURCE_URI="https://github.com/HaikuArchives/BeAE/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="d69a16821f86eda6e94b8297928f9def75f5c750e1e7d6b946d0b195fdc88943"
+SOURCE_DIR="BeAE-$portVersion"
+SOURCE_FILENAME="BeAE-$portVersion.tar.gz"
+
 ARCHITECTURES="x86_gcc2 !x86 ?x86_64"
 SECONDARY_ARCHITECTURES="!x86"
```

Comment:
In case BeAE cannot be easily fixed for x86 and x86_64 we can change SECONDARY_ARCHITECTURES and fix PROVIDES (which is missing two "$secondaryArchSuffix"):
```
SECONDARY_ARCHITECTURES="x86_gcc2"

PROVIDES="
	beae$secondaryArchSuffix = $portVersion
	app:BeAE$secondaryArchSuffix = $portVersion
	"
```

(I've just built beae_x86_gcc2 on x86 and it runs fine ;-)